examples/src/bin/video_player.rs

```diff
@@ -52,20 +52,32 @@ mod imp {
 
         fn class_init(klass: &mut Self::Class) {
             Self::bind_template(klass);
+            klass.install_action(
```

Comment:
We could also consider doing this like for signals with a builder, etc. But that can be done at another time

Actions are in the end very similar to signals.

Reply:
Ah, yes, that can actually work. I will leave that to someone interested in improving how signals can be created/used today.

Reply:
Can you create an issue?